### ISSUE TITLE
One table level on month, week, and day views

### ISSUE THREAD

**Issue by gbonker** [fullcalendar/fullcalendar#6183]
### Checklist

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

For accessibility reasons, it would be wonderful if the month, week, and day views could use only one `<table>`. Right now, there are two `<table>` tags nested inside of each other, which can be very confusing especially for screen reader users.

**From https://fullcalendar.io/demos (v5.5.1):**

![image](https://user-images.githubusercontent.com/6987710/109713960-d8928780-7b6f-11eb-8f4b-839c3fddc6c2.png)

**My team's implementation, for further visual reference:**

![image](https://user-images.githubusercontent.com/6987710/109714194-1abbc900-7b70-11eb-8544-07a1e59e0ee7.png)

#### Background

Users who rely on assistive technology such as screen readers to be aware of the layout and structure of information in a data table are not able to properly interpret the relationship between the rows, cells, the events in the cells and their column headings. Data table headers and data associations MUST NOT be referenced across nested, merged, or separate tables.

People who are blind cannot see the organizational structure of a table with data arranged in rows and columns with corresponding header cells. In order for screen reader users to understand the logical relationships of data arranged in a table, tables need HTML markup that indicates header cells and data cells and defines their relationship. When tables are marked correctly, screen reader users are able to navigate data tables from cell to cell, in a multi-directional way (up, down, right, left), much like navigating a spreadsheet. As they move from cell to cell, screen readers will read the associated header labels.

Tables that are made of nested data tables or multiple data tables that appear as one table break the accessibility of the data presentation as a whole, making it impossible to associate the data and header cells appropriately. It is much better to create one or more simple tables than to create a confusing complex table that technically passes the accessibility guidelines.

Fix this issue by recoding the table as one table marked with proper header cell and data cell relationships. Consider whether you can simplify the table structure.

REFERENCE:
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/tables/

**Comment by acerix** [comment]
I believe this existing ticket captures this need:
https://github.com/fullcalendar/fullcalendar/issues/2536
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.
